src/test/kotlin/model/DirectionTest.kt

```diff
@@ -55,7 +57,11 @@ class DirectionTest : FunSpec({
   }
 
   test("random") {
-    Direction.random().shouldBeInstanceOf<Direction>()
+    generateSequence {
+      Direction.random(Random)
+    }
+      .take(1000)
+      .toSet() shouldBe setOf(N, NE, E, SE, S, SW, W, NW)
   }
```

Comment:
this test is hiperindeterministic xd You rely on random generated seed in the `Random` constuctor